discovery-provider/src/utils/ipfs_lib.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, ipfs_peer_host, ipfs_peer_port, gateway_addresses):
     def get_metadata_from_json(self, metadata_format, resp_json):
         metadata = {}
         for parameter, value in metadata_format.items():
-            metadata[parameter] = resp_json.get(parameter) or value
+            metadata[parameter] = resp_json.get(parameter) if resp_json.get(parameter) != None else value
```

Comment:
does this change behavior?

Reply:
yes! this fixes a bug! I found this issue when I was debugging with Sid last week. as it was before if the initial parameter was any falsey value (e.g. just `False` we would go set it the default from the `metadata_format` schema (usually `None`), which makes it impossible to parse any JSON that has a falsey value from IPFS